### PR TITLE
Replace mui

### PR DESCRIPTION
**Pull request by wendi-yu**
Replace MUI with Antd components

### CODE REVIEW

File: src/components/document/TableView/TableView.jsx [hunk 1]
```diff
@@ -71,41 +72,41 @@ const TableEntry = (sentence) => {
     }
 
     const SentenceVisibilityScale = (props) => {
-        return <Slider {...props} onChange={setSentenceLength} value={sentenceLength} className="w-20 m-2 justify-center items-center flex"/>
+        return <Slider {...props} onChange={setSentenceLength} value={sentenceLength} className="w-20 m-2 justify-center items-center flex" />
```
Comment (RichardYSun) [inline src/components/document/TableView/TableView.jsx:75]: Super not: but there is a space at the end

File: src/components/document/TableView/TableView.jsx [hunk 2]
```diff
@@ -71,41 +72,41 @@ const TableEntry = (sentence) => {
...
         return <div className="flex flex-row p-1">
             <img
-                className={redactionResult==Result.Approved ? "opacity-30" : "opacity-100"}
+                className={redactionResult == Result.Approved ? "opacity-30" : "opacity-100"}
                 src={CheckIcon}
-                onClick={() => {setRedactionResult(redactionResult===Result.Approved ? Result.InReview : Result.Approved)}}/>
+                onClick={() => { setRedactionResult(redactionResult === Result.Approved ? Result.InReview : Result.Approved) }} />
             <img
-                className={redactionResult==Result.Rejected ? "opacity-30" : "opacity-100"}
+                className={redactionResult == Result.Rejected ? "opacity-30" : "opacity-100"}
```
Comment (RichardYSun) [inline src/components/document/TableView/TableView.jsx:85]: I’m guessing you used a lint for this, I missed this but it’s supposed to be “===“ here, can you make the change for me

File: src/components/document/TableView/TableView.jsx [hunk 3]
```diff
@@ -71,41 +72,41 @@ const TableEntry = (sentence) => {
...
+            <img src={CommentIcon} />
+            <img src={NavigateToIcon} />
         </div>
     }
 
-    return <div className={`border border-solid border-slate-100 flex flex-row p-2 ${redactionResult==Result.InReview ? 'opacity-100' : 'opacity-70'}`}>
-        <SentenceWithRedaction className="m-2"/>
-        <SentenceVisibilityScale min={min} max={max}/>
-        <TableActionButtons/>
+    return <div className={`border border-solid border-slate-100 flex flex-row p-2 ${redactionResult == Result.InReview ? 'opacity-100' : 'opacity-70'}`}>
```
Comment (RichardYSun) [inline src/components/document/TableView/TableView.jsx:93]: Same ===

File: src/util/editorCommentUtils.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { v4 as uuid } from "uuid";
-import { Editor, Node } from 'slate'
+import { Editor, Node, Text } from 'slate'
```
Comment (RichardYSun) [inline src/util/editorCommentUtils.js:2]: Is that last import used?

Comment (wendi-yu) [reply src/util/editorCommentUtils.js:2]: Yeah it's used in a `Text.isText` call

**Pull request merged**